### PR TITLE
Added ServerThread class and updated Server and Thread files

### PR DESCRIPTION
**Pull request by chillyssa**
### This pull request includes: 
- Logic to handle the main Server program to listen for a request and then create a new thread for each new client 
- The Thread class has also been started to create a thread for each client submitting requests to the server
- Started building the HTTP responses based on the incoming request from the client
- Worked on the Client program

### Still need: 
- fix the while loop 
- finish response back to client using switch case logic

### CODE REVIEW

File: HW3/TCPHTTPServerThread.java [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+            while((fromClient = cSocketIn.readLine()) != null){
+                // TODO: Read, display to the standard output, and interpret incoming HTTP request message line by line
+                /*
+                * If the method given in the request line is NOT “GET”, it is a “400 Bad Request” case
+                * If the file specified in the URL does not exit/cannot be open, it is a “404 Not Found” case
+                * Otherwise, it is a “200 OK” case
+                */
+                System.out.println(fromClient);
+
+                // switch(status){
```
Comment (S4kRiPp4) [inline HW3/TCPHTTPServerThread.java:44]: i'll try to hammer out this switch statement tomorrow morning

File: HW3/ClientTCPHTTP.java [hunk 2]
```diff
@@ -4,34 +4,114 @@
...
+          // terminal.
+        catch (UnknownHostException e) {
+            System.err.println("Unknown Host: " + host);
+            System.exit(1);
+        } catch (IOException e) {
+            System.err.println("Couldn't get I/O for the connection to: " + host);
+            System.exit(1);
+        }
+        
+        while ((fromUser = sysIn.readLine()) != "N") {
```
Comment (S4kRiPp4) [inline HW3/ClientTCPHTTP.java:49]: will need to see why the server is waiting on a keyboard input before it  moves on to next print statement

**Review by S4kRiPp4 — APPROVED** [review]
looks good i'll try to figure out a solution to the server waiting on input

File: HW3/ClientTCPHTTP.java (hunk 2 above)
Comment (chillyssa) [reply HW3/ClientTCPHTTP.java:49]: fixed this in #15!

**Pull request merged**